cadc-vosi/src/main/java/ca/nrc/cadc/vosi/AvailabilityClient.java

```diff
@@ -89,6 +89,10 @@
     private static Logger log = Logger.getLogger(AvailabilityClient.class);
 
     public static final String AVAILABILITY_ENDPOINT = "/availability";
+    public static final String MIN_DETAIL_PARAMETER = "?detail=min";
+    public static final int DEFAULT_CONNECTION_TIMEOUT = 9000;
+    public static final int DEFAULT_READ_TIMEOUT = 3000;
+    public static final int MIN_DETAIL_MAX_RETRIES = 1;
```

Comment:
For the above timeouts, I'd make connection timeout a little shorter (maybe 6000) and I'd have a small read timeout for detail=min (2000) and a longer one for regular (12000?). 

I don't think I would ever retry on availability checks because the goal is not really to succeed, it is to detect. I hadn't thought of that before, but disabling retry inside HttpGet seems best.

Reply:
Updated the timeout values, and set maxRetries to 1 for all availability checks which will disables any retries.